### PR TITLE
Remove jQuery dependency

### LINKED ISSUE #2: Remove use of jQuery
All jQuery code should be replaced in favour of plain javascript

### PR DESCRIPTION
**Pull request by Pustur**
I tried to fix #2 but I can't test the changes as I'm not a Jira user.  
Hopefully it's a good starting point

### CODE REVIEW

**Comment by arelstone** [comment]
@Pustur this is AWESOME and a very nice enhancement.. Thanks a lot. 👍 ❤️ 

Could you please resolve the conflicts so i can this this out

**Comment by Pustur** [comment]
@arelstone This should do it, feel free to test it now :)

File: index.js [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
-        $('.ghx-swimlane').last().addClass('ghx-last')
-        $('.ghx-swimlane.ghx-first').removeClass('ghx-closed');
-	
-        $("body, div#content, div#ghx-pool").keydown(({ keyCode, preventDefault }) => {
-            preventDefault();
+    onDocumentReady(() => {
+        const swimlanes = Array.from(document.querySelectorAll('.ghx-swimlane'));
+
+        swimlanes.forEach(elem => elem.classList.add('ghx-closed'));
+        swimlanes.slice(-1)[0].classList.add('ghx-last');
```
Comment (arelstone) [inline index.js:19]: I think that `swimlanes` should be treated as an immutable. By using `.splice` you are mutating the array.

You could use some uber cool deconstruction to get the last entry of `swimlanes`

```suggestion
const [, last] = swimlanes;
last.classList.add('ghx-last');
```

**Review by arelstone — CHANGES_REQUESTED** [review]
Really good job. Thanks @Pustur.

I've requested a minor change so be sure that you are not mutating `swimlanes`.

Good job.. Keep it up 👍

File: index.js (hunk 1 above)
Comment (Pustur) [reply index.js:19]: @arelstone I think you misread my code, I used the method `.slice` which does not mutate the array.

Comment (arelstone) [reply index.js:19]: My bad.. Sorry

**Review by arelstone — APPROVED** [review]

**Comment by Pustur** [comment]
Hey @arelstone, may I ask why did you revert the changes? Something didn't work? :(

**Comment by arelstone** [comment]
@Pustur Sorry for not notifying you about the revert, but it was friday afternoon an ii was about to leave work.. The list of excuses is long

Yes, something didn't work. When you pressed the toggling keys nothing happened.

For you to be able to test i've made a html file and pushed it to master. You can find it in [`stub/jira-board.html`](https://github.com/arelstone/jira-swimlane-toggler-chrome-extension/blob/master/stub/jira-board.html). 

You might need to do some changes to the if statement in [`index.js`](https://github.com/arelstone/jira-swimlane-toggler-chrome-extension/blob/master/index.js#L1). 

When you confirmed that things work feel free to reopen the PR and @commentatorboy or I will take a look at it.
This is a really nice enhancement

**Pull request merged**